cmd/kube-apiserver/app/server.go

```diff
@@ -139,6 +139,8 @@ func Run(s *options.APIServer) error {
 	storageFactory, err := genericapiserver.BuildDefaultStorageFactory(
 		s.StorageConfig, s.DefaultStorageMediaType, api.Codecs,
 		genericapiserver.NewDefaultResourceEncodingConfig(), storageGroupsToEncodingVersion,
+		// FIXME: that should be configurable
```

Comment:
It's about making the `GroupVersionResource` overrides being configurable, I'll open a separate issue to address that.